pandas/core/dtypes/base.py

```diff
@@ -352,3 +352,92 @@ def _get_common_dtype(self, dtypes: List[DtypeObj]) -> Optional[DtypeObj]:
             return self
         else:
             return None
+
+
+def register_extension_dtype(cls: Type[ExtensionDtype]) -> Type[ExtensionDtype]:
```

Comment:
Perhaps even their own file?

Right now it might not merit its own file, but in the future I could see us having an entrypoints-based registry, which will add more lines.

Reply:
yeah i agree, let's create registry.py